### PR TITLE
Fix for scheduled events If contact was already removed from campaign

### PR DESCRIPTION
**Pull request by kuzmany**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3929
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
So ugly bug,
Based on my clients issie with scheduled events
and also this issue https://github.com/mautic/mautic/issues/3929
Error log:
```
[  [Doctrine\ORM\EntityNotFoundException]                                                     
  Entity of type 'Mautic\CampaignBundle\Entity\LeadEventLog' for IDs id(xxx) was not found  ](url)
```

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create campaign
2. Create two scheduled action 
- first - Campaign update - remove from this campaign - 1 min delay
- second - Modify tags - 1 min delay
3. Add contact to campaign
4. Run `php app/console m:c:t`
5. Wait 1 min and run `php app/console m:c:t`
6. Return error because triggerScheduledEvents try execute both events, even if first even already removed contact  from campaign

#### Steps to tests:
1. Repeat all steps to reproduce
2. See If works without errors

### CODE REVIEW

**Comment by kuzmany** [comment]
Label: WIP

**Comment by kuzmany** [comment]
Label: Ready for test

File: app/bundles/CampaignBundle/Model/EventModel.php [hunk 1]
```diff
@@ -845,6 +845,7 @@ public function triggerScheduledEvents(
 
             // Get a count
             $events = $repo->getScheduledEvents($campaignId, false, $limit);
+            $this->logger->debug('CAMPAIGN: events #'.print_r($events, true));
```
Comment (escopecz) [inline app/bundles/CampaignBundle/Model/EventModel.php]: @kuzmany do you think you could remove this line? It may cause the log sizes to explode.

Comment (kuzmany) [reply app/bundles/CampaignBundle/Model/EventModel.php]: Removed

**Comment by biggala2310** [comment]
I still have got this same problem. 

[2020-01-15 10:37:19] mautic.WARNING: The reference for contact 1081 and IP address 1261283 is already there. (Unique constraint) [] []
[2020-01-15 11:25:04] mautic.ERROR: CAMPAIGN: A new entity was found through the relationship 'Mautic\CampaignBundle\Entity\LeadEventLog#lead' that was not configured to cascade persist operations for entity: Mautic\LeadBundle\Entity\Lead with ID #. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist  this association in the mapping for example @ManyToOne(..,cascade={"persist"}). [] []


@kuzmany Should I add your code line to 828 row in app/bundles/CampaignBundle/Model/EventModel.php ? I have got only 148 lines in this file. I have Mautic 2.15.3 and PHP Version 7.2.24

**Pull request merged**